libpod/adapter/info_remote.go

```diff
@@ -0,0 +1,56 @@
+// +build remoteclient
+
+package adapter
+
+import (
+	"encoding/json"
+
+	"github.com/containers/libpod/cmd/podman/varlink"
+	"github.com/containers/libpod/libpod"
+)
+
+// Info returns information for the host system and its components
+func (r RemoteRuntime) Info() ([]libpod.InfoData, error) {
+	// TODO the varlink implementation for info should be updated to match the output for regular info
+	var (
+		reply    []libpod.InfoData
+		hostInfo map[string]interface{}
+		store    map[string]interface{}
+	)
+
+	registries := make(map[string]interface{})
+	insecureRegistries := make(map[string]interface{})
+	conn, err := r.Connect()
```

Comment:
Should we put conn inside the LocalRuntime?

Reply:
mheon, I haven't really decided yet because we don't typically make multiple connections.
 usually it is a one-off

Reply:
Ack, fair enough, we can move it over later easy enough if we want to